prow/scripts/cluster-integration/kyma-gke-upgrade-e2e.sh

```diff
@@ -0,0 +1,489 @@
+#!/usr/bin/env bash
+
+# Description: Kyma Upgradeability plan on GKE. The purpose of this script is to install last Kyma release on real GKE cluster, upgrade it with current changes and trigger testing.
+#
+#
+# Expected vars:
+#
+#  - REPO_OWNER - Set up by prow, repository owner/organization
+#  - REPO_NAME - Set up by prow, repository name
+#  - BUILD_TYPE - Set up by prow, pr/master/release
+#  - DOCKER_PUSH_REPOSITORY - Docker repository hostname
+#  - DOCKER_PUSH_DIRECTORY - Docker "top-level" directory (with leading "/")
+#  - KYMA_PROJECT_DIR - directory path with Kyma sources to use for installation
+#  - CLOUDSDK_CORE_PROJECT - GCP project for all GCP resources used during execution (Service Account, IP Address, DNS Zone, image registry etc.)
+#  - CLOUDSDK_COMPUTE_REGION - GCP compute region
+#  - CLOUDSDK_DNS_ZONE_NAME - GCP zone name (not its DNS name!)
+#  - GOOGLE_APPLICATION_CREDENTIALS - GCP Service Account key file path
+#  - MACHINE_TYPE (optional): GKE machine type
+#  - CLUSTER_VERSION (optional): GKE cluster version
+#  - KYMA_ARTIFACTS_BUCKET: GCP bucket
+#  - BOT_GITHUB_TOKEN: Bot github token used for API queries
+#
+# Permissions: In order to run this script you need to use a service account with permissions equivalent to the following GCP roles:
+#  - Compute Admin
+#  - Kubernetes Engine Admin
+#  - Kubernetes Engine Cluster Admin
+#  - DNS Administrator
+#  - Service Account User
+#  - Storage Admin
+#  - Compute Network Admin
+
+set -o errexit
+
+discoverUnsetVar=false
+
+for var in REPO_OWNER REPO_NAME DOCKER_PUSH_REPOSITORY KYMA_PROJECT_DIR CLOUDSDK_CORE_PROJECT CLOUDSDK_COMPUTE_REGION CLOUDSDK_DNS_ZONE_NAME GOOGLE_APPLICATION_CREDENTIALS KYMA_ARTIFACTS_BUCKET BOT_GITHUB_TOKEN; do
+    if [[ -z "${!var}" ]] ; then
+        echo "ERROR: $var is not set"
+        discoverUnsetVar=true
+    fi
+done
+if [[ "${discoverUnsetVar}" = true ]] ; then
+    exit 1
+fi
+
+#Exported variables
+export TEST_INFRA_SOURCES_DIR="${KYMA_PROJECT_DIR}/test-infra"
+export KYMA_SOURCES_DIR="${KYMA_PROJECT_DIR}/kyma"
+export KYMA_SCRIPTS_DIR="${KYMA_SOURCES_DIR}/installation/scripts"
+export TEST_INFRA_CLUSTER_INTEGRATION_SCRIPTS="${TEST_INFRA_SOURCES_DIR}/prow/scripts/cluster-integration/helpers"
+export KYMA_INSTALL_TIMEOUT="30m"
+export KYMA_UPDATE_TIMEOUT="25m"
+export E2E_UPGRADE_TEST_PATH="${KYMA_SOURCES_DIR}/tests/end-to-end/external-solution-integration/chart/external-solution"
+export E2E_UPGRADE_TEST_HELM_TIMEOUT_SEC=10000 # timeout in sec for helm operation install/test
+export E2E_UPGRADE_TEST_TIMEOUT_SEC=600 # timeout in sec for e2e upgrade test pods until they reach the terminating state
+export E2E_UPGRADE_TEST_NAMESPACE="external-solution-e2e-upgrade-test"
```

Comment:
(nit): `external-solution-upgrade-e2e-test` is more consistent with the other occurences of `post-kyma-tests-end-to-end-upgrade-e2e`